tests/features/views/list-view/view-layout.feature

```diff
@@ -33,11 +33,11 @@ Scenario Outline: Venue display in List View
 
 # While the defined default is to only show the venue title, city, and state/province, users should be able to customize their list view template to show more or less venue information (e.g. address, country)
 
-Scenario: Automatic event excerpt (35 words) in List View
+Scenario: Automatic event excerpt (30 words) in List View
     Given the next public, published upcoming event has the following event description: "Pie apple pie cookie chocolate gingerbread jujubes gingerbread cotton candy chocolate. Chocolate bar dragée lollipop caramels. Sweet tootsie roll gummi bears fruitcake jelly beans caramels candy gummi bears. Lollipop jelly beans jelly-o brownie pudding macaroon liquorice bear claw dessert. Cupcake ice cream marshmallow bear claw tart dragée gummi bears cake. Topping sugar plum topping tiramisu jujubes. Caramels ice cream chupa chups apple pie. Muffin danish gingerbread fruitcake. Cookie candy apple pie cupcake cupcake liquorice. Cheesecake sweet roll gummies apple pie."
```

Comment:
Can we add one more `Given` line about using V2 of views? Since this should only be for the new Views.

Reply:
Done

Reply:
Done